browser/profiles/brave_profile_impl.cc

```diff
@@ -56,7 +58,7 @@ BraveProfileImpl::BraveProfileImpl(
     notification_registrar_.Add(this, chrome::NOTIFICATION_PROFILE_DESTROYED,
                                 content::Source<Profile>(parent_profile));
     base::PostTaskAndReply(
-        FROM_HERE, {base::ThreadPool()}, base::DoNothing(),
+        FROM_HERE, {content::BrowserThread::UI}, base::DoNothing(),
```

Comment:
We'd best use modern API
`To post tasks to the main thread or to the IO thread, use content::GetUIThreadTaskRunner({}) or content::GetIOThreadTaskRunner({}) from content/public/browser/browser_thread.h`

https://chromium.googlesource.com/chromium/src.git/+/master/docs/threading_and_tasks.md#posting-to-the-main-thread-or-to-the-io-thread-in-the-browser-process

Reply:
Addressed by a457702908ba76e12186cd553211567040b6309b